### PR TITLE
Fix: GrapesJS assets transfer is slowing admin panel view loading

### PR DESCRIPTION
**Pull request by patrykgruszka**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ Y ]
| New feature/enhancement? (use the a.x branch)      | [ N ]
| Deprecations?                          | [ N ]
| BC breaks? (use the c.x branch)        | [ N ]
| Automated tests included?              | [ Y ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 

#### Description:
I noticed that all GrapesJS assets are sent with view requests. In this case, if you have a lot of assets, it can significantly slow down view loading.

In my case, with large assets database the Contact, Companies, Segments, Campaigns views are transferring > 2MB of GrapesJS data.
image
image

This PR changes this behavior and loads assets asynchronously when GrapesJS builder is initialized. 

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open ten test GrapesJS builder including assets 
3. GrapesJS assets should not be transferred in panel view requests as encoded JSON, it can be verified using browser dev tools:
image

### CODE REVIEW

**Comment by patrykgruszka** [comment]
@cla-bot check

**Review by jos0405 — DISMISSED** [review]
Hi, it does what it claims.
The assets are not visible in panel anymore

File: plugins/GrapesJsBuilderBundle/Controller/FileManagerController.php [hunk 1]
```diff
@@ -37,4 +37,17 @@ public function deleteAction()
 
         return $this->sendJsonResponse(['success'=> true]);
     }
+
+    /**
+     * @return JsonResponse
+     */
+    public function assetsAction()
```
Comment (escopecz) [inline plugins/GrapesJsBuilderBundle/Controller/FileManagerController.php]:
```suggestion

    public function assetsAction(): JsonResponse
```

Comment (escopecz) [reply plugins/GrapesJsBuilderBundle/Controller/FileManagerController.php]: Please create a functional test that will hit this endpoint and check that some pre-installed asset is present in the response.

**Review by escopecz — CHANGES_REQUESTED** [review]
The code looks great! Please create one functional test that will cover the new endpoint.

**Review by mabumusa1 — DISMISSED** [review]
This PR is good to go, I opened 10 tabs with grapejs opened, I looked in the developers there were assets that are passed all the time, after applying the PR they were not loaded anymore.

Thanks @patrykgruszka for this PR

**Comment by patrykgruszka** [comment]
@escopecz  done, added functional test and suggestion. Thanks everyone for tests!

**Review by escopecz — APPROVED** [review]
Thank you!

**Pull request merged**